### PR TITLE
Improvements: process additional fields

### PR DESCRIPTION
**Pull request by lciolecki**
- Add createdAt and createdBy fields at process
- Add hsql and psql migrations

### CODE REVIEW

File: engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala [hunk 1]
```diff
@@ -5,28 +5,34 @@ import pl.touk.nussknacker.ui.security.api.Permission.Permission
 
 sealed trait LoggedUser {
   val id: String
-  def can(category: String, permission: Permission): Boolean
+  val username: Option[String]
   val isAdmin: Boolean
+  def can(category: String, permission: Permission): Boolean
+  def getName: String = username.getOrElse(id)
```
Comment (witekw) [inline engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala]: It's a trick to save something in not null db column. I would move it closer to the database not to be reused.

Comment (arkadius) [reply engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala]: Indeed @lciolecki can we have `username` as a field as it was before and fill it with the same value in situation when username is the same as id?

File: ui/server/src/main/resources/db/migration/hsql/V1_026__ProcessCreatedAtByFields.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+-- create fields
+ALTER TABLE "processes" ADD COLUMN "created_at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP;
```
Comment (arkadius) [inline ui/server/src/main/resources/db/migration/hsql/V1_026__ProcessCreatedAtByFields.sql]: Good job :+1:

File: engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala (hunk 1 above)
Comment (lciolecki) [reply engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala]: Ofc I can :)

**Pull request merged**